gemac/txEngine.py

```diff
@@ -1,24 +1,180 @@
-from myhdl import block, instances
+from myhdl import block, always_seq, Signal, intbv
+
+crc32_tab = (
```

Comment:
I had a discussion about it with @tompdillon . If i compute crc using generator polynomial it creates unnecessary complexity compared to which using table driven approach seems much better. I also worried about the large table but Tom said ROM is cheaper in FPGA and to go with simpler complexity, it enables us to work directly on bytes compared to working on bits while not using the table.